cms/wizards/urls.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from django.conf.urls import patterns, url
+
+from .views import WizardCreateView
+
+
+urlpatterns = patterns('',  # NOQA
+    url(r'^create/$',
+        WizardCreateView.as_view(), name='wizard_create'),
```

Comment:
I highly recommend either using namespaced urls when including this or changing the url name to ``cms_wizard_create`` this is because ``wizard_create`` is too generic and can easily conflict with other apps